### PR TITLE
Merge change 010

### PR DESCRIPTION
**Pull request by asaxsma**
This is the item we conditionally approved at change 009 meeting.  We finally approved it online.  Calling this change 010.

### CODE REVIEW

File: as_pacmod.dbc [hunk 1]
```diff
@@ -771,6 +775,36 @@ BO_ 1536 WATCHDOG_RPT: 8 PACMOD
...
  SG_ MINI_SYSTEM_PRESENT_FAULT : 62|1@0+ (1,0) [0|1] ""  CUSTOMER_ECU
 
+BO_ 316 BRAKE_DECCEL_CMD: 8 CUSTOMER_ECU
+ SG_ ENABLE : 0|1@0+ (1,0) [0|1] ""  PACMOD
+ SG_ IGNORE_OVERRIDES : 1|1@0+ (1,0) [0|1] ""  PACMOD
+ SG_ CLEAR_OVERRIDE : 2|1@0+ (1,0) [0|1] ""  PACMOD
+ SG_ BRAKE_DECCEL_CMD : 15|16@0+ (0.001,0) [0|10] "m/s^2"  PACMOD
+ SG_ XBR_EBI_MODE : 25|2@0+ (1,0) [0|2] ""  PACMOD
+ SG_ XBR_PRIORITY : 27|2@0+ (1,0) [0|3] ""  PACMOD
+ SG_ XBR_CONTROL_MODE : 29|2@0+ (1,0) [0|2] ""  PACMOD
```
Comment (JWhitleyWork) [inline as_pacmod.dbc:785]: I thought the final decision was to split up the DECCEL_CMD and the XBR bits into a _CMD and an _AUX_CMD? Did I misunderstand the outcome?

**Review by JWhitleyWork — CHANGES_REQUESTED** [review]
See comment on DECCEL_CMD message.

**Comment by asaxsma** [comment]
I was thinking we came down on the side of combining them.  It avoided having to invent this new "aux_command".  Does anyone else remember where we came down at?

**Comment by asaxsma** [comment]
Here's what I remember about the conversation:
Pro for combining them: didn't need to invent new message, coupling with deccel command is pretty tight.
Pro for separating them: didn't have to violate existing cmd message architecture (like steering does).

**Comment by mlemm99** [comment]
I remember combining them because they are so dependant. The mode dictates if the Accel command is accepted

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by JWhitleyWork** [comment]
I think Mike might be right. LGTM.

**Review by JWhitleyWork — DISMISSED** [review]

**Comment by mlemm99** [comment]
LGTM = Let's Get To Merging

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Review by driscoll85 — DISMISSED** [review]

**Review by mlemm99 — APPROVED** [review]

**Review by driscoll85 — APPROVED** [review]

**Review by JWhitleyWork — APPROVED** [review]

**Pull request merged**